src/app/test-record/test-record.component.html

```diff
@@ -1,4 +1,142 @@
-<p>
-  <span>JSON response:</span>
-  {{ testType | json }}
-</p>
+<div class="govuk-width-container">
+  <a class="govuk-back-link" id="test-back-button" vtmBackButton>Back</a>
+
+  <main class="govuk-main-wrapper">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-full">
+        <h1 class="govuk-heading-xl test-heading" id="test-heading">Test record</h1>
+      </div>
+    </div>
+
+    <div class="govuk-grid-row">
+      <ng-container *ngIf="!editState">
+        <div class="govuk-grid-column-one-quarter">
+          <button
+            id="test-change-test-record"
+            class="govuk-button govuk-button--secondary"
+            data-module="govuk-button"
+            (click)="switchCurrentState('edit')"
+          >
+            Change test record
+          </button>
+        </div>
+      </ng-container>
+
+      <ng-container *ngIf="editState">
+        <div class="govuk-grid-column-one-quarter">
```

Comment:
Could probably be a separate component as this is reused for vo / create/edit views